lib/matplotlib/projections/polar.py

```diff
@@ -39,6 +39,18 @@ def __init__(self, axis=None, use_rmin=True,
         self._use_rmin = use_rmin
         self._apply_theta_transforms = _apply_theta_transforms
 
+    def __str__(self):
+        return ("{}(\n"
+                    "{},\n"
+                    "    use_rmin={},\n"
```

Comment:
I have a slightly preference for this string not to be indented, or remove the explicit spaces and call indent_str for use_rmin too (either way).  The idea was to make the source layout "look like" the final result.  Same comment throughout.